libraries/adaptive-expressions/adaptive/expressions/function_utils.py

```diff
@@ -234,3 +247,17 @@ def set_property(instance: object, property: str, val: object):
         value = result
         error = None
         return value, error
+
+    @staticmethod
+    def try_parse_list(value: object, out_list: list):
+        islist = False
+        out_list.clear()
+        if isinstance(value, list):
+            for i in list(value):
+                out_list.append(i)
+            islist = True
+        return islist
```

Comment:
this function is actually not necessary. just use isinstance(value, list) to check if it is list. This function is only useful in C# library

Reply:
fixed